CareKit/CareCard/OCKCareCardViewController.m

```diff
@@ -130,6 +130,10 @@ - (void)prepareView {
     if (!_headerView) {
         _headerView = [[OCKCareCardTableViewHeader alloc] initWithFrame:CGRectZero];
     }
+    if ([_headerTitle length] == 0) {
+        _headerTitle = OCKLocalizedString(@"CARE_CARD_HEADER_TITLE", nil);
+    }
+    _headerView.title = _headerTitle;
```

Comment:
Change to `_headerView.title = self.headerTitle;`

This way the headerTitle length check only needs to be done in the setter.
Sorry for not catching this earlier. Thanks!